pandas/tests/io/test_pytables.py

```diff
@@ -5167,3 +5168,75 @@ def test_dst_transitions(self):
                 store.append('df', df)
                 result = store.select('df')
                 assert_frame_equal(result, df)
+
+
+class TestReadPyTablesHDF5(Base):
+    """
+    A group of tests which covers reading HDF5 files written by plain PyTables
+    (not written by pandas).
+    """
+
+    def _create_simple_hdf5_file_with_pytables(self):
```

Comment:
is this necessary or could you just add a file in pandas\tests\io\data

Reply:
> is this necessary or could you just add a file in pandas\tests\io\data

Oh, that's some early feedback. Thank you!

I think a proper test for the fix for #11188 should confirm that the minimal working example for reproducing the problem does not fail anymore.

I also think this is more readable (when the test itself shows how the data file was created, in contrast to committing a binary file of which it's not 100 % clear and discoverable of how it has been created).

Also, I'd like to think of this (using the `PyTables` API to create an HDF5 file) as an important aspect of this test: it effectively confirms a common way to use `PyTables` and `pandas` in tandem; `PyTables` for writing an HDF5 file, and `pandas` for reading it. These tests are not so much about the contents in the file, but about that a specific way to create the file matches a specific way to parse the file.

Well, let's see about the "fix", looking forward to your feedback there. I think this will be more controversial.